### ISSUE TITLE
application won't show up, no error, always debug-like

### ISSUE THREAD

**Issue by GiForce** [kivy/kivy#4714]
Hi
I installed dependencies and kivy following your homebrew/pip section (pip in a fresh created virtualenv). Bash still didn't recognise "kivy", so I downloaded the Kivy3-app, copied it as "Kivy" to Applications and made the symlink. Kivy is here....(yupieh)..
When starting an kivy-app, nothing happens but the output of terminal is always "debug"-style, 
[debug-terminal-output](https://gist.github.com/GiForce/59f56b6359bc8b7642d766c8a01f4483.js)
means it's the same with log_level = info. The .txt file in the App's content is exactly the same. Also when "import kivy" in the python/kivy shell, the first 3 [info] lines appear.
I didn't found that behaviour mentioned, so I wonder what I'm missing?
Os X 10.12.1

**Comment by GiForce** [comment]
Excuse me. It works! As you may knew already :) Kivy app's obviously don't run as __main__. Silly me, the correct shebang for Osx I'd worked out already.... Guys, your documentation is great. It's so properly styled it almost hurt my eyes ;)

**Comment by GiForce** [comment]
That's still  not true, I proceed the "Pong Game" tutorial and if __name__== '__main__': is used, it works. I tried my previous (hello world -like) examples again with this, and they work too. Yet.
There is nothing else  I changed. I'm so not an expert, and won't even try :)

**Comment by GiForce** [comment]
hello. even noone ever reads here. As I suspected for a while, I installed kivy double. My first attempt with pip wasn't recognised. Anyway I wiped out the pip install now, and still everything works. That means I never had a real "debug" output, because I edited the pip-installed kivy config.  I feel a little dumb o.x 
Even I'd prefer a pip in my virtualenv for playing around. I'm happy it works. Maybe the homebrew libs are also double? Ew, I won't mess around anymore. Eventually, Kivy is so nice I will probably use it as my prefered Gui-framework. 👾
